### DISCUSSION TITLE
Versions supporting Sitecore 8.2?

### DISCUSSION THREAD

**Discussion by SorenKerrn** [SitecorePowerShell/Console#1249] [Q&A] [answered]
I looked at the release pages and can tell that SPE 5.1 and 6.0 seems to support Sitecore 8.2.
How about v. 6.1, 6.2 and 6.3?

**Comment by michaellwest** [comment, marked as answer]
Hey @SorenKerrn if you check out the official documentation we have it all listed.

https://doc.sitecorepowershell.com/appendix

![image](https://user-images.githubusercontent.com/933163/145074605-24d4edfe-773c-4516-95c4-73bcfb524f87.png)
